lib/signs_ui/signs/sign.ex

```diff
@@ -59,7 +72,7 @@ defmodule SignsUI.Signs.Sign do
       id: sign_id,
       config: %{
         mode: :headway,
-        expires: config["expires"]
+        expires: expiration_from_string(config["expires"])
```

Comment:
By our JSON "spec" `config["expires"]` is allowed to be `nil` (meaning no expiration), so `expiration_from_string/1` should have a typespec of `String.t() | nil`, and another function clause that handles that.

Reply:
Done